plugins/wasi_ocr/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+wasmedge_add_library(wasmedgePluginWasiOCR
+  SHARED
+  wasiocrenv.cpp
+  wasiocrfunc.cpp
+  wasiocrmodule.cpp
+)
+
+target_compile_options(wasmedgePluginWasiOCR
+  PUBLIC
+  -DWASMEDGE_PLUGIN
+)
+
+if(CMAKE_SYSTEM_NAME MATCHES "Darwin")
+  target_link_options(wasmedgePluginWasiOCR
```

Comment:
We no longer need to link these options, please refer the other plugin for the examples.

Reply:
Fixed accordingly in the next commit.